redbeams/src/main/java/com/sequenceiq/redbeams/service/network/SubnetChooserService.java

```diff
@@ -40,4 +41,10 @@
         }
         return subnetSelectionResult.getResult();
     }
+
+    public List<CloudSubnet> chooseSubnetForPrivateEndpoint(Collection<CloudSubnet> subnetMetas, DBStack dbStack, boolean existingNetwork) {
+        NetworkConnector networkConnector = cloudPlatformConnectors.get(new CloudPlatformVariant(dbStack.getCloudPlatform(), dbStack.getPlatformVariant()))
+                .networkConnector();
+        return networkConnector.chooseSubnetsForPrivateEndpoint(subnetMetas, existingNetwork).getResult();
```

Comment:
pls. add a log line

Reply:
Added a log line that lists all subnets suitable for private endpoint.